<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>谐波减速器数据采集系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>178</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>通讯配置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portNameBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudrateBox">
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataBitsBox">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>校验位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parityBox">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopBitsBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>数据库配置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>数据库IP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dataBaseIP">
         <property name="text">
          <string>localhost</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>数据库：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dataBaseName">
         <property name="text">
          <string>harmonic</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userName">
         <property name="text">
          <string>root</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>21</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passWord">
         <property name="text">
          <string>zk123</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="connectDB">
         <property name="text">
          <string>数据库连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectDB">
         <property name="text">
          <string>数据库断开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>试验设置</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QRadioButton" name="motorRadioButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>电机实验</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="motorBox">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>#1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="hrRadioButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>减速器实验</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="HRBox">
        <item>
         <property name="text">
          <string>#1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="beltRadioButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>皮带实验</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="beltBox">
        <item>
         <property name="text">
          <string>#1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>#3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <property name="spacing">
       <number>21</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="startCollectButton">
        <property name="text">
         <string>开始采集</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopCollectButton">
        <property name="text">
         <string>停止采集</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>7</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>233</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>基本信息</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>17</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>电机型号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motorCategory">
         <property name="text">
          <string>多摩川HDXXX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>减速器型号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="harmonic">
         <property name="text">
          <string>绿的谐波</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>17</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>皮带型号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="motorCategory_2">
         <property name="text">
          <string>XXXX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>641</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>实时曲线</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QCustomPlot" name="c_motorCurrentplot"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="c_motorAngleplot"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="c_motorVelocityplot"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>工作日志：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QListWidget</extends>
   <header location="global">qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
